src/main/java/org/kohsuke/github/ServiceDownException.java

```diff
@@ -0,0 +1,17 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+
+/**
+ * Special {@link IOException} case for http exceptions, when {@link HttpException} is thrown due to GitHub service
+ * being down.
+ *
+ * @implNote Inherits from {@link HttpException} to not break compatibility with existing code base.
```

Comment:
`site` doesn't like `@implNote`. 

```suggestion
 *
 * Inherits from {@link HttpException} to maintain compatibility with existing clients.
```